pkgs/by-name/cu/cubyz/package.nix

```
@@ -0,0 +1,121 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  callPackage,
+  makeWrapper,
+  libx11,
+  libxcursor,
+  libGL,
+  alsa-lib,
+  vulkan-loader,
+  vulkan-validation-layers,
+  vulkan-tools,
+  zig,
+}:
+
+let
+  # This is kinda atrocious, but it works
+  # Override default zig flags
+  zig_hook =
+    (zig.overrideAttrs {
+      version = "0.15.0";
+      src = fetchFromGitHub {
+        owner = "ziglang";
+        repo = "zig";
+        rev = "bd97b66186dabb3533df1ea9eb650d7574496a59";
+        hash = "sha256-EVIg01kQ3JCZxnnrk6qMJn3Gm3+BZzPs75x9Q+sxqBw=";
+      };
+    }).hook.overrideAttrs
+      {
+        zig_default_flags = "";
+      };
+in
+
+stdenv.mkDerivation (finalAttrs: {
+  version = "0.0.1";
+  pname = "cubyz";
+  src = fetchFromGitHub {
+    owner = "pixelguys";
+    repo = "cubyz";
+    tag = finalAttrs.version;
+    hash = "sha256-8dceQWoTUJHkClKiFH573gvi94SmTSiSl42Rh9+sPoE=";
```

Works great for me, just had to change the hash for 0.0.1.

```suggestion
    hash = "sha256-SbMRr4kktwagYUyVBKFZLOwgSmkPngV8NbwkJRk2Zvg=";
```

Starting the game gives a brief error
```sh
❯ Cubyz
/run/current-system/sw/bin/Cubyz: line 51: cd: /home/theaninova/.config/cubyz: No such file or directory
```

Which is solved by simply creating the directory (is that just an issue with the game?)